starboard/shared/starboard/memory.cc

```diff
@@ -102,27 +102,15 @@ void* SbMemoryAllocateChecked(size_t size) {
 namespace {
 
```

Comment:
SbMemoryAllocated is deprecated in SB16.

Reply:
Right - so this change is a no-op. Are you suggesting i leave the #ifdefs in here and just remove the config from gn/md files ?

Reply:
I guess the question is do you want to potentially affect SB15/SB14 implementations if partners rebase and rebuild, but don't change the SB version?

Reply:
I backed out the .cc changes on the off chance that someone somewhere used it. It had no effect in our build configs.